app/controllers/FileReadyController.scala

```diff
@@ -47,18 +46,27 @@ class FileReadyController @Inject() (
   def onPageLoad(): Action[AnyContent]                               = (identify andThen profileAuth andThen getData andThen requireData).async {
     implicit request =>
       (for {
-        Some(downloadDataSummary) <- downloadDataConnector.getDownloadDataSummary(request.eori)
+        Some(downloadDataSummary) <-
```

Comment:
you need to handle the options (see the data-store pr for examples) or you could just remove the options